src/libguac/file-private.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#ifndef GUAC_FILE_PRIVATE_H
+#define GUAC_FILE_PRIVATE_H
+
+/**
+ * Functions used by the internals of "file.h".
+ *
+ * @file file-private.h
+ */
+
+/**
+ * Tests whether the given string is a filename with no other path components
+ * present.
+ *
+ * @param filename
+ *     The filename to check.
+ *
+ * @return
+ *     Zero if the provided string is a filename without any other path
+ *     components, non-zero if at least one path component is present.
```

Comment:
I think this is actually backward from how the function is implemented - looking at the below code in `file.c`, it looks like it returns `0` if a path separate *is* found, and `1` if a path separator is *not* found. Which makes more sense to me given the function name `guac_is_filename()`.

Reply:
Ack! You're right. I'll fix that.